lib/matrix_client.ex

```diff
@@ -5,28 +5,42 @@ defmodule MatrixClient do
 
   alias MatrixClient.{Session}
   alias MatrixSDK.{Client}
+  alias MatrixSDK.Client.{Request}
```

Comment:
Small thing: these `{}` (and above) aren't actually needed 

```suggestion
  alias MatrixSDK.Client.Request
```